src/Frame/HybiFrame.php

```diff
@@ -131,13 +133,7 @@ public function encode(string $payload, int $type = Protocol::TYPE_TEXT, bool $m
      */
     protected function generateMask()
     {
-        if (\extension_loaded('openssl')) {
-            return \openssl_random_pseudo_bytes(4);
-        } else {
-            // SHA1 is 128 bit (= 16 bytes)
-            // So we pack it into 32 bits
-            return \pack('N', \sha1(\spl_object_hash($this).\mt_rand(0, \PHP_INT_MAX).\uniqid('', true), true));
-        }
+        return \random_bytes(4);
```

Comment:
Is this change strictly necessary for the fix? If not, I'd rather it be moved to a separate atomic PR.

Reply:
moved to https://github.com/chrome-php/wrench/pull/22